app/src/main/res/values/styles.xml

```diff
@@ -1,10 +1,11 @@
 <resources>
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
It will change in future for day/night mode, explore this theme for day/night mode.